client/src/features/sessionsV2/SessionStartPage.tsx

```diff
@@ -257,72 +258,36 @@ function StartSessionFromLauncher({
   launcher,
   project,
 }: StartSessionFromLauncherProps) {
-  const {
-    containerImage,
-    defaultSessionClass,
-    isFetchingOrLoadingStorages,
-    resourcePools,
-    startSessionOptionsV2,
-  } = useSessionLauncherState({
+  const hasCustomQuery = new URLSearchParams(location.search).has("custom");
```

Comment:
`"custom"` should probably be defined as a constant. Also, this should use a hook to get the query params:

```suggestion
  const [searchParams] = useSearchParams(); // from "react-router-dom-v5-compat"
  const hasCustomQuery = searchParams.has("custom");
```